tests/ra_aid/proc/test_windows_compatibility.py

```
@@ -0,0 +1,65 @@
+    def test_create_process(self):
+        """Test process creation on Windows."""
+        with patch('subprocess.Popen') as mock_popen:
+            mock_process = MagicMock()
+            mock_process.returncode = 0
+            mock_popen.return_value = mock_process
+            
+            proc, _ = create_process(['echo', 'test'])
+            
+            assert mock_popen.called
+            args, kwargs = mock_popen.call_args
+            assert kwargs['stdin'] == subprocess.PIPE
```

Ruff (0.8.2)

Undefined name `subprocess`

(F821)